.github/workflows/staging-tests.yml

```diff
@@ -12,7 +12,7 @@ on:
     branches:
       - main
   schedule:
-    - cron: '0 */4 * * *'
+    - cron: '0 */8 * * *'
```

Comment:
NB: I changed this to run every 8 hours instead of 4, under the reasoning that 8 hours corresponds roughly to 3 different working times each day and gives us enough time between failures to actually institute a fix without too many duplicates piling up.